src/ott/geometry/costs.py

```diff
@@ -206,6 +205,7 @@ def pairwise(self, x: jnp.ndarray, y: jnp.ndarray) -> float:
   def h_transform(
       self,
       f: Callable[[jnp.ndarray], float],
+      x_init: Optional[jnp.ndarray] = None,
```

Comment:
Sequence[jnp.ndarray]?

Reply:
Change the API, now the `x_init` is directly expose in `f_h` with the kwargs.